### ISSUE TITLE
Incompatibilities with five.pt

### ISSUE THREAD

**Issue by rodfersou** [plone/plone.app.imaging#16]
As we notice into these issues (https://github.com/zopefoundation/five.pt/issues/5 and https://github.com/malthe/chameleon/issues/59) we have some incompatibilities with `five.pt` and `chameleon`.

In summary, `plone.app.imaging` is waiting for a `list` while `chameleon` uses a `tuple` instead as an speed optimization.

In this issue, we should make `plone.app.imaging` accept a `tuple` as an argument.

**Comment by hvelarde** [comment]
I confirm the issue only happens when using the traversal variant; so, this will fail:

```html
<img tal:replace="structure context/@@images/image/mini" />
```

but this works:

```html
<img tal:define="scales context/@@images"
     tal:replace="structure python: scales.tag('image', scale='mini')"
     />
```

**Comment by mauritsvanrees** [comment]
Fixed in 1.0.12 and 2.0.1.

**Comment by rodfersou** [comment]
@mauritsvanrees thank you very much
